<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgEnvironment</class>
 <widget class="QDialog" name="CQDlgEnvironment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Environment Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Colors / Fog</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqBackgroundColorUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Background (up)</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqBackgroundColorDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Background (down)</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqAmbientLightColor">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>25</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Ambient light</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqFogAdjust">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>45</y>
      <width>190</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust fog parameters</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Calculation preprocessing</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum triangle size (absolute) [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqMaxTriangleSize">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>25</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqSaveCalcStruct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>401</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Save operation also saves existing calculation structures</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum triangle size (relative)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqMinRelTriangleSize">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>45</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>401</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>See also variable 'triCountInOBB' in 'system/usrset.txt'</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Various</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqShapeTexturesDisabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>391</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Shape textures disabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqNextSaveIsDefinitive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>391</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Lock scene after next scene save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqExtensionString">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>65</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Extension string</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Scene content acknowledgments / Info</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPlainTextEdit" name="qqAcknowledgments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqBackgroundColorUp</tabstop>
  <tabstop>qqBackgroundColorDown</tabstop>
  <tabstop>qqAmbientLightColor</tabstop>
  <tabstop>qqFogAdjust</tabstop>
  <tabstop>qqMaxTriangleSize</tabstop>
  <tabstop>qqMinRelTriangleSize</tabstop>
  <tabstop>qqSaveCalcStruct</tabstop>
  <tabstop>qqShapeTexturesDisabled</tabstop>
  <tabstop>qqNextSaveIsDefinitive</tabstop>
  <tabstop>qqExtensionString</tabstop>
  <tabstop>qqAcknowledgments</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
